### DISCUSSION TITLE
[Weekly Meeting] [2024-10-18] Meeting Minutes

### DISCUSSION THREAD

**Discussion by porcelli** [apache/incubator-kie-issues#1548] [Weekly Meetings]
- Current state of CI
  - DDoS?
    - setup settings.xml config
    - when building had to point maven to previous repos... however the settings was also shared with nightlies and this was releasing (cascading!!!) to all repos during those nightlies
    - now only a single failure, only proxy error not time out anymore... 
  - python failures
    - disabled but might not be fixed
    - but it might be still happening - @baldimir noticed recently
    - pip is limiting packages installed
       - `broken?` needs to be added, but option can't be found as an option
       - we are not sure.... might be a repo by repo issue (due GHA slow rollout of the upgrade)
- Apache KIE 10
  - RC2 blocker
  - RC3
    - Currently blocked by https://github.com/apache/incubator-kie-issues/issues/1534 - in progress (with failures) by @mcruzdev 
    - Once blocked is solved, we'll start running the updated pipelines
       - due large changes needed, we might face issues that needs to be adjusted

**Comment by mcruzdev** [comment]
Hi @porcelli, in addition to this, we have a list that we still need to resolve (there are more projects to remove wrappers, etc.).

**Reply by porcelli** [reply]
Please add more details to the issue.

**Reply by mcruzdev** [reply]
Thank you, [done](https://github.com/apache/incubator-kie-issues/issues/1534#issuecomment-2423881883)!

**Comment by dmitry-buykin** [comment]
BTW, images in docker are not reflecting RC1/RC2 releases:
https://hub.docker.com/r/apache/incubator-kie-sandbox-extended-services/tags (some main, updated 22 days ago)
https://hub.docker.com/r/apache/incubator-kie-sandbox-webapp/tags (has 10.0.0)
https://hub.docker.com/r/apache/incubator-kie-cors-proxy/tags (some main, updated 22 days ago)

So it's quite hard to test the future 10.x version with online editor, while vscode extensions are not updated.

**Reply by tiagobento** [reply]
Please see my comment below, but notice that we won't be publishing Release Candidate artifacts to Docker Hub nor any other public registry. Release Candidates for Apache KIE 10.0.0 will be published to https://dist.apache.org/repos/dist/dev/incubator/kie/ exclusively.

**Comment by tiagobento** [comment]
@dmitry-buykin Indeed we do have some wrong tags pushed to Docker Hub :/ I compiled a list a while ago and we need to do some cleanup there. https://gist.github.com/tiagobento/120e3646c11f23c975ff898dadf929b4
